src/_stylesheets/_grid.scss

```diff
@@ -83,4 +83,4 @@
 
 .app-grid__cell--image-full-width img {
   width: 100%;
-}
+}
```

Comment:
Pithy comment that this now doesn't have an extra blank line at the end. This an old boring tech thing https://dev.to/documendous/why-you-should-end-your-source-files-with-a-new-line-156g. Not critical for the operation of the site.